jvb/src/main/java/org/jitsi/videobridge/Conference.java

```diff
@@ -1174,6 +1181,12 @@ public JSONObject getDebugState(boolean full, String endpointId)
     {
         JSONObject debugState = new JSONObject();
         debugState.put("id", id);
+
+        if (!isRtcStatsEnabled)
```

Comment:
This would also prevent us from manually debugging such conferences. How about including the `isRtcStatsEnabled` flag in the output and having the rtcstats pusher skip conferences with the flag set?

Reply:
Will do!

Reply:
https://github.com/jitsi/jvb-rtcstats-push/pull/8